bedrock/firefox/views.py

```diff
@@ -526,3 +524,23 @@ def ios_testflight(request):
     return l10n_utils.render(request,
                              'firefox/testflight.html',
                              {'newsletter_form': newsletter_form})
+
+
+class FirefoxHubView(BlogPostsView):
+    blog_posts_limit = 1
+    blog_posts_template_variable = 'articles'
+    blog_slugs = 'firefox'
+    blog_tags = ['home']
+    template_name = 'firefox/hub/home.html'
+
+    # Is this necessary with a redirect added to firefox/redirects.py?
+    # (The redirect in firefox/redirects.py doesn't work atm, so...)
+    def dispatch(self, request, *args, **kwargs):
```

Comment:
I wouldn't override `dispatch`, I'd do `get`. It's not a big deal, but that would let the View class handle POST requests differently (in that they'd get a 405). The could would be roughly the same though.